CONTRIBUTING.md

```diff
@@ -4,5 +4,5 @@ To help you get started on the right track, check out the following resources to
 
 - The [Article Style Guide](https://new.ppy.sh/wiki/Article_Style_Guide) for the styling standards in wiki.
 
-- The [Getting Started Guide](https://new.ppy.sh/wiki/osu!wiki_contribution_guide) _(New to the GitHub workflow?, you should read this first)_.
+- The [Getting Started Guide](https://new.ppy.sh/wiki/osu!wiki_contribution_guide) _(New to the GitHub workflow? You should read this first)_.

```

Comment:
if they login new.ppy.sh and select language before, osu-web will redirect to their language version

so, it's not neccessary to write it here (and save words(see L1, L2))